content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,452 @@
+
+To serve your application to the browser, `cd` into the project root directory and run the command:
+```bash
+ng serve 
+```
+
+### Create a component
+Now that we've setup our project, let's proceed and create a `todo` component to handle all our logics:
+
+Run the command below to create a `todo` component:
+```bash
+ng generate component todo
+# OR
+ng g c todo
+```
+
+Next, let's add template code in the `todo.component.html` file. Again, we will use tailwind for styling. 
+
+> Note: Tailwind CSS [CDN link](https://v1.tailwindcss.com/docs/installation#using-tailwind-via-cdn) should be added to the `index.html` file.
+
+Copy and paste the code below in the `todo.component.html` file.
+```html   
+    <div class="h-100 w-full flex items-center justify-center bg-teal-lightest font-sans">
+        <div class="bg-white rounded shadow p-6 m-4 w-full lg:w-3/4 lg:max-w-lg">
+            <div class="mb-4">
+                <h1 class="text-grey-darkest">Todo List</h1>
+                <form class="flex mt-4">
+                    <input class="shadow appearance-none border rounded w-full py-2 px-3 mr-4 text-grey-darker" placeholder="Add Todo">
+                    <button class="flex-no-shrink p-2 border-2 rounded text-teal border-teal hover:text-white hover:bg-teal">Add</button>
+                </form>
+            </div>
+            <div>
+                <div class="flex mb-4 items-center">
+                    <p class="w-full text-grey-darkest">task 1</p>
+                    <button class="flex-no-shrink p-2 ml-4 mr-2 border-2 rounded hover:text-white text-green border-green hover:bg-green">Done</button>
+                    <button class="flex-no-shrink p-2 ml-2 border-2 rounded text-red border-red hover:text-white hover:bg-red">Remove</button>
+                </div>
+                <div class="flex mb-4 items-center">
+                    <p class="w-full text-green">task 2</p>
+                    <button class="flex-no-shrink p-2 ml-4 mr-2 border-2 rounded hover:text-white text-green border-green hover:bg-green">Done</button>
+                    <button class="flex-no-shrink p-2 ml-2 border-2 rounded text-red border-red hover:text-white hover:bg-red">Remove</button>
+                </div>
+            </div>
+        </div>
+    </div>
+```
+
+Once you paste the HTML code, you should see something like the image below on your browser.
+
+![Todo List](/engineering-education/how-to-build-a-task-manager-with-angular/todo-list.png)
+
+> Note: The buttons are static, so we will make them work next.
+
+###  Add Tasks to the To-do list
+Tasks will be added to the to-do list by clicking the `Add` button or by pressing the `Enter` key.
+
+Let's see what happens under the hood;
+Since the `Add` button and the input box are in a form element, we handle them using the Angular [FormGroup](https://angular.io/api/forms/FormGroup#description) property.
+
```

Comment:
Could explain why we're using the FormGroup and not Formbuilder?  Give us more details about this.